docs/pages/server-access/guides/openssh.mdx

```diff
@@ -340,6 +337,87 @@ authenticate the host via the certificate we generated earlier.
 
 </Details>
 
+<Details title="Using Trusted Clusters?">
+
+If you are using Trusted Clusters, the `tsh config` command will print an OpenSSH
+client configuration block for the root cluster and all currently known leaf clusters.
+
+This configuration block uses a `ProxyCommand` directive to connect to SSH 
+servers in a leaf cluster through the root cluster's Proxy Service.
+
+In scenarios where Proxy Service instances in a leaf cluster are reachable by
+SSH clients, you might prefer to connect directly through the leaf proxies to
+achieve lower latency.
+
+To support this scenario, the `tsh proxy ssh` command provides the `--jumphost`/`-J` flag.
+This flag points to the public address of a cluster's Proxy Service and indicates 
+that the OpenSSH client will be connecting through that proxy instead of the proxy
+you're currently logged into.
+
+```txt
+Host *.{{ .NodeName }}.{{ .ClusterName }}
+   Port 3022
+   ProxyCommand tsh proxy ssh -J proxy.leaf1.example.com:443 %r@%h:%p
+```
+
+When the `-J` flag is used with the `{{proxy}}` template variable, the jump
+proxy address and host server address can be dynamically determined using proxy
+templates defined in the `tsh` configuration file (`~/.tsh/config/config.yaml`
+or a global `/etc/tsh.yaml`)
+
+```txt
+Host *.example.com
+   Port 3022
+   ProxyCommand tsh proxy ssh -J {{proxy}} %r@%h:%p
+```
+
+```yaml
```

Comment:
This YAML snippet needs some introduction. I would add a paragraph that transitions from the last snippet. For example:

```markdown
To configure the way `tsh` evaluates the `{{proxy}}` template variable when
using the `-J` flag, add a `proxy_templates` field to your `tsh` configuration
file (`~/.tsh/config/config.yaml` or `/etc/tsh.yaml` for all users on your system):
```
We could then delete the mention of the `tsh` configuration file location from above.

Again, if this configuration only applies to the `-J` flag, I'm guessing we need some explicit instructions for the user to edit their `tsh config`-created configuration to include the `-J` flag. Also, we should probably introduce the `-J` flag outside of this guide, since it looks like it's intended for all `tsh` commands, not just those involving legacy OpenSSH servers.

Reply:
> Also, we should probably introduce the `-J` flag outside of this guide, since it looks like it's intended for all `tsh` commands, not just those involving legacy OpenSSH servers.

We currently mention the `-J` flag in one other spot - https://goteleport.com/docs/ver/10.0/server-access/guides/tsh/#ssh-jump-host

And then there are there's the ssh man pages which are relevant as well:

```
     -J destination
             Connect to the target host by first making a ssh connection to the jump host described by destination and then establishing
             a TCP forwarding to the ultimate destination from there.  Multiple jump hops may be specified separated by comma charac‐
             ters.  This is a shortcut to specify a ProxyJump configuration directive.  Note that configuration directives supplied on
             the command-line generally apply to the destination host and not any specified jump hosts.  Use ~/.ssh/config to specify
             configuration for jump hosts.
```

Do you think we need to expand the tsh jump host section? If so, how will we fit in these docs to include the OpenSSH client integration with `tsh config`? Maybe that's where an `OpenSSH Client guide` would come in?